unused/Han_Xiognu_war.md

```diff
@@ -0,0 +1,10 @@
+Wanting to clear away this threat to the Han people, Liu Bang, the founder of the Han dynasty, led an army to attack the Xiognu. The Xiognu at first appeared to retreat, that, however turned out to be a feint. The Xiognu suddenly turned and attacked the Han—at the Battle of Baideng. The Han, having fallen for the trap, suffered a devastating defeat. Han Emperor Liu Bang barely managed to escape to a nearby fortress, which was in turn besieged by the Xiognu. The newly founded empire was on the brink of collapse.
+But the Han were able to bribe the Shanyu's own wife into convincing the latter to break off the siege and sign a peace treaty instead. This peace treaty obligated the Han to send yearly tribute payments to the Xiognu in order to keep the peace. Although these payments were very high, the Han,  seeing no other choice, paid for the next several decades, even though the Xiognu still frequently plundered Han territory.
+It was Emperor Wu who finally dared to break the peace. He raised a giant army and planned an ambush at the city of Mayi. A trader was sent to lure the Xiognu into the trap by claiming he had killed the city's ruler and the Shanyu could take it. However, meeting hardly any resistance, the Xiognu got suspicious and eventually caught wind of the plan by questioning a captured Han soldier. The Xiognu turned around and, even though the Han tried to pursue, were able to withdraw without losing a single man. 
+The two were now officially at war. At first, the Han seemed to stand no chance, as their infantry and chariots proved to be too slow compared to the Xiognu cavalry. And the Chinese had no experience in fighting a nomadic enemy.
+Over the next few years, the Xiognu would conduct large-scale raids, and the Han were trying to defend themselves as well as possible: They extended and upgraded the Great Wall to counter the Xiognu's hit-and-run tactics. The Han military also shifted focus towards cavalry, raising a cavalry force of hundreds of thousands. For that, general Li Guangli even conducted a whole separate war against a distant kingdom—the War of the Heavenly Horses.
+And slowly the tides began turning: Xiognu raids were defended well, and Han commanders like Wei Qing and Huo Qubing were celebrated as heroes (among the Han, of course) for conducting successful counter-raids. They managed to take over the Hexi Corridor, a long, narrow passage leading through an otherwise ungracious and hostile terrain. Controlling it meant being able to trade or tax trade with the Western parts of the world. The Xiognu had lost a crucial support pillar for their economy. Eventually, they retreated northwards behind the Gobi Desert. And the Han, they got back to doing what they did best: building more walls. Extending the existing wall structure along the Hexi Corridor enabled them to open direct trade with the West.
```

Comment:
```suggestion
And slowly the tides began turning: Xiognu raids were fought off, and Han commanders like Wei Qing and Huo Qubing conducted successful counter raids. The Han managed to take over the Hexi Corridor, a long, narrow passage leading through an otherwise ungracious and hostile terrain. Controlling it meant being able to trade with or tax trader from the Western parts of the world. The Xiognu had lost a crucial support pillar for their economy. Bit by bit, they retreated northwards past the Gobi Desert. And the Han built more walls, extending the existing wall structure along the Hexi Corridor, enabled them to open direct trade with the West.
```